src/v2/Apps/Auctions/Components/TrendingLots/TrendingLots.tsx

```diff
@@ -0,0 +1,100 @@
+import React from "react"
+import { createFragmentContainer, graphql } from "react-relay"
+import { useTracking } from "react-tracking"
+
+import { Box, Shelf, Spacer, Text } from "@artsy/palette"
+import { AuthContextModule, clickedArtworkGroup } from "@artsy/cohesion"
+
+import { extractNodes } from "v2/Utils/extractNodes"
+import { ShelfArtworkFragmentContainer } from "v2/Components/Artwork/ShelfArtwork"
+import { tabTypeToContextModuleMap } from "../../Utils/tabTypeToContextModuleMap"
+import { TrendingLots_viewer } from "v2/__generated__/TrendingLots_viewer.graphql"
+import { useAnalyticsContext } from "v2/System"
+export interface TrendingLotsProps {
+  viewer: TrendingLots_viewer
+}
+
+const TrendingLots: React.FC<TrendingLotsProps> = ({ viewer }) => {
+  const { trackEvent } = useTracking()
+  const { contextPageOwnerType } = useAnalyticsContext()
+  const contextModule = tabTypeToContextModuleMap.trendingLots as AuthContextModule
+
+  const nodes = extractNodes(viewer.trendingLotsConnection)
+
+  if (nodes.length === 0) {
+    return null
+  }
+
+  return (
+    <>
+      <Box>
+        <Text as="h3" variant="lg" color="black100">
+          Trending lots{" "}
+          <sup>
+            <Text as="span" variant="xs" color="brand">
+              {/* @ts-expect-error STRICT_NULL_CHECK */}
```

Comment:
Presumably: `viewer.trendingLotsConnection?.edges?.length` would be safe — should avoid including these `ts-expect-error`s

Reply:
Removed!